core/core-configuration-layer.el

```diff
@@ -201,6 +198,26 @@ LAYER has to be installed for this method to work properly."
     ("gnu"   . "elpa.gnu.org/packages/"))
   "List of ELPA archives required by Spacemacs.")
 
+(defvar configuration-layer--elpa-archives-status
+  '(("melpa" . unknown)
+    ("org" . unknown)
+    ("gnu" . unknown))
+  "Status of ELPA archives.")
+
+(defvar configuration-layer-fallback-package-archives
```

Comment:
I wonder if we really need this. I mean, fallback archives can be added to the end of `configuration-layer--elpa-archives`, so instead of switching to values from `configuration-layer-fallback-package-archives` we can just remove first associated url and search for another. Once no urls are available for specific archive - error!